### ISSUE TITLE
Pytorch From Scratch Path

### ISSUE THREAD

**Issue by Wei-HaiMing** [Strawberry64/Pareidolia-Desktop#69]
**Issuer:** Armando Vega

 **Date:** 04-24-2026

 **Description:** Create a path in the Pytorch pipeline to allow for the layers chosen by the user to be built into the Pytorch model. Use existing ImageDataModule and modify it to dynamically change the mean and std normalization based on the model project type (scratch or pretrained). 

 **Length of work / Timeline:** 1-2 Days
